lib/challenge.js

```diff
@@ -33,6 +44,11 @@
 var pet_owner;
 
 // Answer Starts Here
+pet_owner (date,vaccination){
+  date: "10-10-15"
+  vaccination: "Rabies vaccination"
+  }
```

Comment:
This is a mix of function literal notation and object literal notation. Partial credit.